docs/benchmarks/_index.md

```diff
@@ -127,9 +127,9 @@ How to setup the environment for the Kafka Performance Test.
 
     ```bash
     kubectl exec -it kafka-test -n kafka bash
-    ./opt/kafka/bin/kafka-topics.sh --zookeeper zookeeper-client.zookeeper:2181 --topic perftest --create --replication-factor 3 --partitions 3
-    ./opt/kafka/bin/kafka-topics.sh --zookeeper zookeeper-client.zookeeper:2181 --topic perftest2 --create --replication-factor 3 --partitions 3
-    ./opt/kafka/bin/kafka-topics.sh --zookeeper zookeeper-client.zookeeper:2181 --topic perftest3 --create --replication-factor 3 --partitions 3
+    ./opt/kafka/bin/kafka-topics.sh --bootstrap-server kafka-all-broker.kafka:29092 --topic perftest --create --replication-factor 3 --partitions 3
```

Comment:
Question: for Koperator alone, shouldn't the service name be `kafka-headless`?

Reference: https://github.com/banzaicloud/koperator/blob/master/pkg/resources/kafka/kafka.go#L184-L198

Reply:
Depends whether kafka-headless is enabled or not.
There are multiple `KafkaCluster` samples where it is disabled, so the kafka-all-broker service is safer IMO, because it is always available.

Reply:
I got a bit unsure about whether in case of headless the all-broker-service is there, let me check later when I have time.

Reply:
You were right, it depends on the `headlessService` `KafkaCluster` field.
So this needs to be diverged into 2 commands, I will do it later.

Reply:
Fixed in https://github.com/banzaicloud/koperator-docs/compare/31795bdead34cfa09405d0bebabd2b51ef4a9c02..a15f6899ad0a667008c6e3be09643dcd67fcddb9.